src/orderbook/base.jl

```diff
@@ -0,0 +1,41 @@
+abstract type AbstractOrderBook{Tprice, Tvolume} <: AbstractPriceData{Tprice, Tvolume} end
+
+struct OrderBookException <: Exception
+    msg::String
+end
+
+function price(ob::AbstractOrderBook, volume::Volume; raise=false)
```

Comment:
Maybe I should have wrote something like

    price(ob::AbstractOrderBook{Tprice, Tvolume}, volume::Tvolume; raise=false) where {Tprice, Tvolume}

as I said it's a very old code and I wasn't very fluent using parametric types

Reply:
I'd rather use `volume::Real` 
Unless it has hidden implication?